recipes/picohttpparser/all/conanfile.py

```diff
@@ -0,0 +1,41 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, apply_conandata_patches, export_conandata_patches
+from conan.tools.cmake import CMakeToolchain, CMake
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.51.1"
+
+class PicohttpparserConan(ConanFile):
+    name = "picohttpparser"
+    description = "tiny HTTP parser written in C (used in HTTP::Parser::XS et al.)"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/h2o/picohttpparser"
+    topics = ("http", "parser")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
```

Comment:
```suggestion
        cmake_layout(self, src_folder="src")
```

for cmake project, use cmake_layout